lib/form526_backup_submission/utilities/convert_to_pdf.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'mini_magick'
+
+module Form526BackupSubmission
+  module Utilities
+    class ConvertToPdf
+      CAN_CONVERT = ['.jpg', '.jpeg', '.png', '.gif', '.bmp', '.txt'].freeze
```

Comment:
Could you use shorthand format like this: `%w[.jpg .jpeg ... .txt]` here?

Reply:
Yea... I could, and usually rubocop makes me/does that for me. Not sure why it did not in this instance. But I will change it.